CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0] - 2024-08-23
+
+### Changed
+
+- Upgraded `locidex/merge` to version `0.2.2` and updated `input_assure.py` and test data for compatibility with the new `mlst.json` allele file format.
```

Comment:
Can you change to version `0.2.3` for locidex here.

Reply:
Fixed here: [288ce4d](https://github.com/phac-nml/arboratornf/pull/19/commits/288ce4d1619b9b55aada24b152507466cc4cc12d)